### PR TITLE
adding business duration receipe

### PR DESCRIPTION
**Pull request by frankzhu93**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by frankzhu93** [comment]
@conda-forge/help-python The receipe has been passed all the checks and ready for review. Thanks.

**Comment by frankzhu93** [comment]
@conda-forge/help-python The receipe has been passed all the checks and ready for review. Let me know if there is any issue need to be fixed. Thanks.

File: recipes/business-duration/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{% set name = "business-duration" %}
+{% set version = "0.66" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://files.pythonhosted.org/packages/e5/1d/12b3bb612a19d20ca2d9fe7ebfc80ab6b9e1d5d575c6a3dcd121c7cbfed4/business_duration-0.66.tar.gz"
```
Comment (xhochy) [inline recipes/business-duration/meta.yaml:9]: The URL should be templated with the version number.

File: recipes/business-duration/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
```
Comment (xhochy) [inline recipes/business-duration/meta.yaml:25]: Please add a call to pip check here

File: recipes/business-duration/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    - business_duration
+
+about:
+  home: "https://github.com/gnaneshwar441/Business_Duration"
+  license: "MIT"
+  license_family: "MIT"
+  license_file: "PKG-INFO"
```
Comment (xhochy) [inline recipes/business-duration/meta.yaml:32]: This doesn't sound like a typical license file.

**Review by xhochy — COMMENTED** [review]
I would recommend to regenerate this recipe with grayskull to work around some issues.

File: recipes/business-duration/meta.yaml (hunk 3 above)
Comment (dopplershift) [reply recipes/business-duration/meta.yaml:32]: It's not, it's only the license metadata from packaging, no actual text with language of the license.

**Pull request closed without merging**